cuprite.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.files         = Dir["{lib}/**/*"] + %w[LICENSE README.md]
 
-  s.required_ruby_version = ">= 2.3.0"
+  s.required_ruby_version = ">= 2.4.0"
```

Comment:
What was the reason to require 2.4? I think in general 2.3 is still in broad use, so we might avoid doing it.